src/js/modules/accounts/components/SignInPage.js

```diff
@@ -1,22 +1,68 @@
 import React from "react";
 import { connect } from "react-redux";
 import { bindActionCreators } from "redux";
-import { SignInForm, SignUpForm } from "./forms";
+import { Link } from "react-router-dom";
+import { Grid, Row, Col } from "react-bootstrap/lib";
+import injectSheet from "react-jss";
 
-import { signIn, signUp } from "../actions";
+import { SignInForm } from "./forms";
+import { signIn } from "../actions";
 
-const SignInPage = ({ signIn, signUp }) => {
+const styles = {
```

Comment:
I'd consider factoring this out, since you have the exact same code in two files. Not sure where it should go though